about.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="body">
+        <div class="socials">
+            <li><a href=""><img src="images/facebook.png" alt="facebook"></a></li>
+            <li><a href=""><img src="images/twitter.png" alt="twitter"></a></li>
+            <li><a href="">English</a></li>
+            <li><a href="">My page</a></li>
+            <li><a href="">Log Out</a></li>
+        </div>
+    <header>
+        <nav class="mobile-nav">
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#my-courses">Courses</a></li>
+                <li><a href="#">About</a></li>
+            </ul>
+            <a href=""><img src="images/x-menu.png" alt="logo" width="20px" height="20px"></a>
+        </nav>
+        <nav class="desktop-nav">
+            <a href="#"><img src="images/logo.png" alt="x-menu" width="50px" height="50px"></a>
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#my-courses">Courses</a></li>
+                <li><a href="#">About</a></li>
+            </ul>
+            <div class="mobile-menu-icon">
+              <a href="#">
+                <img src="images/menu.svg" alt="logo" width="30px" height="30px">
+              </a>
+            </div>
+        </nav>
+    </header>
+    <h2 class="welcome-about-us">
+        Welcome to Course-R Droid, where learning becomes fun!
+    </h2>
+    <div class="information">
+        Course-R Droid is a technical school where you get to learn any form of technology that interest you from the hardware to the software part of any technological system.
+    </div>
+    <p class="contact-p">Please contact us via email for further questions about enrollment procedures at oguzieibehrichard@gmail.com.</p>
+    <div class="about-us-logo">
+        <img src="images/logo.png" alt="logo">
+        <p>Course-R Droid was founded by a young student who is currently the CEO of this great institution Mr Richard Oguzie-Ibeh, He and his friends started this establishment from a very young age of 19.</p>
+    </div>
```

Comment:
You did wel implement this section in your about section. But you need to fix it the description should be at he top then the logo after the descrition

This is your design: 

![Screen Shot 2022-10-18 at 11 12 00](https://user-images.githubusercontent.com/79506105/196389322-8402562e-537b-442e-a3a9-6425ab7580e5.png)

Your design should look similar to this:




![About page 2](https://user-images.githubusercontent.com/79506105/196389732-ce4432cc-4689-4575-b3de-aec818c22543.png)
This should also be done in your mobile view.

Reply:
Sure boss 😃 👍 , I'll do that right now 💯